### PR TITLE
Feature sign up flow url filter

### PR DESCRIPTION
**Pull request by aghimpu**
https://github.com/teamforus/forus/issues/1097

### CODE REVIEW

File: src/sources/forus-platform/pug/tpl/pages/sign-up.pug [hunk 1]
```diff
@@ -260,6 +260,23 @@
                 +card_header(7)
 
                 .card-section.card-section-primary
+                    .row.form   
+                        .col.col-lg-6
+                            .form-label Organizations
```
Comment (maxvisser) [inline src/sources/forus-platform/pug/tpl/pages/sign-up.pug]: translate to: Organisaties

**Comment by maxvisser** [comment]
labels can be added to i18n

File: src/sources/forus-platform/js/angular-1/components/SignUpComponent.js [hunk 2]
```diff
@@ -271,8 +271,47 @@ let SignUpComponent = function(
...
+            });
+
+            $ctrl.fundOrganizations = $ctrl.fundOrganizations.map(fundOrganization => {
+                fundOrganization.id_str = fundOrganization.id += '';
+
+                return fundOrganization;
+            });
+
+            $ctrl.fundOrganizations.unshift({ id_str: 'null', name: 'All Organizations' });
+            $ctrl.fundLabels.unshift({ key: 'null', name: 'All Labels' });
```
Comment (maxvisser) [inline src/sources/forus-platform/js/angular-1/components/SignUpComponent.js]: Translate: Alle labels

File: src/sources/forus-platform/js/angular-1/components/SignUpComponent.js [hunk 3]
```diff
@@ -271,8 +271,47 @@ let SignUpComponent = function(
...
+                processedLabels.push(fund.label.id);
+            });
+
+            $ctrl.fundOrganizations = $ctrl.fundOrganizations.map(fundOrganization => {
+                fundOrganization.id_str = fundOrganization.id += '';
+
+                return fundOrganization;
+            });
+
+            $ctrl.fundOrganizations.unshift({ id_str: 'null', name: 'All Organizations' });
```
Comment (maxvisser) [inline src/sources/forus-platform/js/angular-1/components/SignUpComponent.js]: Translate: Alle organisaties

File: src/sources/forus-platform/js/angular-1/i18n/i18n-nl.js [hunk 4]
```diff
@@ -534,6 +534,10 @@ module.exports = {
             url_text: "Download de app <i>Me</i> op uw mobiele telefoon via de link:",
             url_address: "www.forus.io/DL"
         },
+        filters: {
+            all_organizations: 'All organizations',
```
Comment (maxvisser) [inline src/sources/forus-platform/js/angular-1/i18n/i18n-nl.js]: Alle organisaties

File: src/sources/forus-platform/js/angular-1/i18n/i18n-nl.js [hunk 5]
```diff
@@ -534,6 +534,10 @@ module.exports = {
...
+            all_labels: 'All labels'
```
Comment (maxvisser) [inline src/sources/forus-platform/js/angular-1/i18n/i18n-nl.js]: Alle labels

**Review by maxvisser — CHANGES_REQUESTED** [review]

**Review by maxvisser — APPROVED** [review]

**Comment by maxvisser** [comment]
@dev-rminds backend is also ready for review?

**Comment by dev-rminds** [comment]
> @dev-rminds backend is also ready for review?

Ready for testing

**Comment by maxvisser** [comment]
fund_id does work. I see it was changed from FundId to fund_id; I like this change but it needs to still work ofcourse.

**Pull request merged**